src/applications/vaos/reducers/newAppointment.js

```diff
@@ -665,12 +677,15 @@ export default function formReducer(state = initialState, action) {
           initialSchema,
         );
       } else {
+        const systems = action.parentFacilities.filter(
```

Comment:
I'm assuming that CC requests should go to a root system, not just a parent facility